### PR TITLE
Added a file index.js

### PR DESCRIPTION
**Pull request by sajjanrajvaidya**
@joshskkim @BHamp @sfshaider 
Can you please check this?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+// THIS IS A GIT TEST
```
Comment (BHamp) [inline index.js:1]: dope

Comment (sfshaider) [reply index.js:1]: dope

Comment (sfshaider) [reply index.js:1]: dope

**Comment by joshskkim** [comment]
dope

**Pull request merged**